js/index.js

```diff
@@ -16,3 +16,28 @@ for (let i = 0; i < skills.length; i++ ){
     skill.innerText = skills[i];
     skillsList.appendChild(skill);
 };
+
+const messageForm = document.querySelector('form[name=leave_message]');
+messageForm.addEventListener('submit', (event) => {
+    event.preventDefault();
+    var name = event.target.name.value;
+    var email = event.target.email.value;
+    var message = event.target.message.value;
+    console.log(name);
+    console.log(email);
+    console.log(message);
+    var messageSection = document.querySelector('#messages');
+    var messageList = messageSection.querySelector('ul');
+    var newMessage = document.createElement('li');
+        newMessage.innerHTML = '<a href="mailto:' + email + '"><span id="email"></span></a>'
```

Comment:
```suggestion
    newMessage.innerHTML = `<p>${message}</p><a href="mailto:${email}">${name}</a>`;
```
Or something like that. You just need to expand the template to include all the information you have.
(Also keep the indentation the same as the line above: The code runs either way, but it will be easier to read if you stick to conventions.)